packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -198,6 +208,21 @@ export class StatsAnalyzer extends EventsScope {
     const videoReceiver = cloneDeep(emptyVideoReceive);
     const videoShareReceiver = cloneDeep(emptyVideoReceive);
 
+    if (this.isMultistream) {
```

Comment:
@brycetham  This approach makes changing the `multistreamEnabled` clearer than passing it to all utility functions. We also have some calculations for MQE data not only with the utility function. This results in the need to pass and change all instances of "audioSender, audioShareSender, audioReceiver, audioShareReceiver, videoSender, videoShareSender, videoReceiver, videoShareReceiver" with the value of `this.isMultistream.` I don't feel good about its way of leaking logic.

Reply:
Hmm in my opinion, I still think it makes sense to pass it to the MQA util functions. Even though it does require tweaking the params for all of them (that's more of a consequence of these functions written poorly which we'll fix in the refactor), I like it better for a few reasons:

1. All the logic related to populating the MQE is in the util functions, so it's good to have them all in the same place.
2. In the refactor, we'll want to pass `isMultistream` into the MQE builders anyways (though we'd do this in the constructor which is much cleaner). So having it in the util functions would more closely align with what we'd end up doing in the refactor.

Reply:
@brycetham updated with the discussed version, it looks good to me, please tell me how you feel about that